src/sections/HeroImage.tsx

```diff
@@ -1,7 +1,17 @@
 import { Button } from "@/components/ui/button";
+import ArrowRight from "@/assets/icons/ArrowRight";
+import Discord from "@/assets/icons/DiscordFill";
 import React from "react";
 
 const HeroImage = () => {
+  const handleDiscordRedirect = () => {
```

Comment:
For Future reference: This is bad practice since it reloads the page, but in future days we will implement react router and put a link inside the button or style the link as a button.